xroad-catalog-collector/CHANGELOG.md

```diff
@@ -0,0 +1,64 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [0.10.0.1] - 2020-02-14
```

Comment:
Is this (0.10.0.1) the version number of the package? I it reflected in the RPM package name also?

Reply:
I got this version number from the RPM packages from S3.

Reply:
Checking the release bucket: https://s3.console.aws.amazon.com/s3/object/xroad-release/xroad-catalog/rhel/7/stable/xroad-catalog-lister-0.10.0-1.20180205111242git3814f03.el7.x86_64.rpm?region=eu-west-1&tab=overview
There is this package from year 2018. 
xroad-catalog-lister-0.10.0-1.20180205111242git3814f03.el7.x86_64.rpm

So I think we should now increase the rpm package version number,  since there are changes to the version 0.10. I don't remember exactly where the version number change is to be put. Probably at least to build_rmp.sh

Maybe this commit helps:

https://github.com/vrk-kpa/xroad-catalog/commit/4db168fee14baed41b5a3dddfb98db1f89912035#diff-d2955185ece0abe85c3f8dfc34497e7f